src/ChainManagerRegistry.php

```diff
@@ -51,7 +51,7 @@ public function getManagers(): array
         return \array_reduce(
             $this->managerRegistries,
             static function(array $carry, ManagerRegistry $managerRegistry) {
-                return \array_merge($carry, $managerRegistry->getManagers());
+                return \array_merge($carry, \array_values($managerRegistry->getManagers()));
```

Comment:
`ManagerRegistry::getManagers()` actually return `array<string, ObjectManager>`, where the key is the objectManager's name. So there is a risk of overlay.

This actually makes our method return `list<ObjectManager>` but the interface says `@return ObjectManager[]` so we don't break the contract...